relay.go

```diff
@@ -193,18 +196,21 @@ func (r *Relayer) Relay(f *Frame) error {
 
 // Receive receives frames intended for this connection.
 func (r *Relayer) Receive(f *Frame, fType frameType) {
-	{
-		// TODO: Since this block is only checking for safety, we should not
-		// enable this in production builds.
-
-		// If we receive a response frame, we expect to find that ID in our outbound.
-		// If we receive a request frame, we expect to find that ID in our inbound.
-		items := r.receiverItems(fType)
+	// If we receive a response frame, we expect to find that ID in our outbound.
+	// If we receive a request frame, we expect to find that ID in our inbound.
+	items := r.receiverItems(fType)
+	isOriginator := fType == responseFrame
 
-		if _, ok := items.Get(f.Header.ID); !ok {
-			r.logger.WithFields(
-				LogField{"ID", f.Header.ID},
-			).Warn("Received a frame without a RelayItem.")
+	item, ok := items.Get(f.Header.ID)
+	if !ok {
+		r.logger.WithFields(
+			LogField{"ID", f.Header.ID},
+		).Warn("Received a frame without a RelayItem.")
```

Comment:
Yes, probably. The existing code just logged a warning and forwarded the frame, and I didn't want to change any more than necessary.

Happy to make that change, though.